lib/tailwind/src/components/templates/arrayTemplate.tsx

```diff
@@ -0,0 +1,43 @@
+import { z } from "zod";
+import { useArrayTemplate, useRenderTemplate } from "../..";
+import { generateInitialData } from "@nanhtu/utils";
+import { ErrorsListTemplate } from "./errorsListTemplate";
+import React from "react";
+
+export const ArrayTemplate: React.FC<{
+  schema: z.ZodArray<any>;
+  path: string[];
+  liveValidate?: boolean;
+}> = ({ schema, path, liveValidate }) => {
+  const { value, errors, addItem, moveItem, removeItem } = useArrayTemplate(
+    path,
+    () => generateInitialData(innerSchema),
+  );
+  const RenderTemplate = useRenderTemplate();
+  const innerSchema = schema.element;
+
+  return (
+    <div>
+      {errors && <ErrorsListTemplate errors={errors} />}
+      {Array.isArray(value) && value.length > 0 ? (
```

Comment:
this condition should wrap in to variable like
```
const isValidValue = Array.isArray(value) && value.length > 0;
```

Reply:
update